newrelic/hooks/mlmodel_openai.py

```diff
@@ -12,15 +11,83 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import openai
+
 import uuid
+
+import openai
+
 from newrelic.api.function_trace import FunctionTrace
-from newrelic.common.object_wrapper import wrap_function_wrapper
-from newrelic.api.transaction import current_transaction
 from newrelic.api.time_trace import get_trace_linking_metadata
-from newrelic.core.config import global_settings
+from newrelic.api.transaction import current_transaction
 from newrelic.common.object_names import callable_name
-from newrelic.core.attribute import MAX_LOG_MESSAGE_LENGTH
+from newrelic.common.object_wrapper import wrap_function_wrapper
+from newrelic.core.config import global_settings
+
+
+def wrap_embedding_create(wrapped, instance, args, kwargs):
+    transaction = current_transaction()
+    if not transaction:
+        return
```

Comment:
I fixed this in the async PR but this should be:
```suggestion
        return wrapped(*args, **kwargs)
```